testplan/runnable/base.py

```diff
@@ -126,6 +128,9 @@ def get_options(cls):
             ConfigOption(
                 'pdf_style', default=defaults.PDF_STYLE,
                 block_propagation=False): Style,
+            ConfigOption(
+                'http_url', default=None,
+                block_propagation=False): Or(str, None),
```

Comment:
Where does this configuration need to propagate to exactly?

Reply:
HTTPExporter  works like WebServerExporter, in `get_default_exporters` will check this argument in config, then initialized with url=config.http_url

Reply:
Sure, I remember that WebServerExporter needed to propagate its configuration somewhere too I just couldn't quite remember the details.